src/com/facebook/buck/ide/intellij/IjProjectCommandHelper.java

```diff
@@ -233,6 +251,75 @@ private TargetGraph getProjectGraphForIde(
         buckEventBus, cell, enableParserProfiling, executor, passedInTargets);
   }
 
+  @SuppressWarnings("unchecked")
+  private ExitCode copyGeneratedGoSource(TargetGraphAndTargets targetGraphAndTargets)
+      throws IOException, InterruptedException {
+    HashMap<String, BuildTargetSourcePath> generatedPackages = new HashMap<>();
+    ClassLoader classLoader = pluginManager.getPluginClassLoader("com.facebook.buck.features.go");
+    try {
+      Class<?> goLibraryDescriptionArgClass =
+          Class.forName("com.facebook.buck.features.go.GoLibraryDescriptionArg", true, classLoader);
+      for (TargetNode<?, ?> targetNode : targetGraphAndTargets.getTargetGraph().getNodes()) {
+        Object constructorArg = targetNode.getConstructorArg();
+        if (constructorArg.getClass() == goLibraryDescriptionArgClass) {
+          Method getPackageNameMethod = goLibraryDescriptionArgClass.getMethod("getPackageName");
+          Optional<String> packageName =
+              (Optional<String>) getPackageNameMethod.invoke(constructorArg);
+          if (packageName.isPresent()) {
+            generatedPackages.putAll(
```

Comment:
Should we remove some line breaks from these chaining calls? (same relates to some other statements below)

Reply:
Those line breaks are added by Google Java formatter, in order to pass CI checks